config/serverless.es.yml

```diff
@@ -13,6 +13,21 @@ enterpriseSearch.enabled: false
 xpack.fleet.enabled: false
 xpack.observabilityAIAssistant.enabled: false
 
+## Fine-tune the search solution feature privileges.
+xpack.features.overrides:
+  ### Dashboards feature is moved from Analytics category to the Search one.
+  dashboard.category: "enterpriseSearch"
+  ### Dev Tools feature is moved from Analytics category to the Search one.
+  dev_tools.category: "enterpriseSearch"
+  ### Discover feature is moved from Analytics category to the Search one.
+  discover.category: "enterpriseSearch"
+  ### Machine Learning feature is moved from Analytics category to the Management one.
+  ml.category: "management"
+  ### Stack Alerts feature is moved from Analytics category to the Search one renamed to simply `Alerts`.
+  stackAlerts:
+    name: "Alerts"
```

Comment:
__note:__ I'm leaning towards removing all `name` overrides for the time being since our i18n system doesn't support i18n IDs provided via configs (or even constants). What do you think?